src-backend/api/generator.py

```diff
@@ -54,22 +54,22 @@ def __init__(self, element):
         self.element = element
 
         if not self.element.element_type:
-            self.__raise_error('Element with id %d has no type')
+            self.__raise_error('Element on page %d has no type')
```

Comment:
Make this more verbose and say `Error on page id:% d:%`

The more debug info the better